system/gatekeeper/templates/constrainttemplate-region-value-mismatch.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: templates.gatekeeper.sh/v1
+kind: ConstraintTemplate
+metadata:
+  name: gkregionvaluemismatch
+spec:
+  crd:
+    spec:
+      names:
+        kind: GkRegionValueMismatch
+      validation:
+        # Schema for the `parameters` field
+        openAPIV3Schema:
+          type: object
+          properties:
+            helmManifestParserURL:
+              type: string
+            region:
+              type: string
+
+  targets:
+    - target: admission.k8s.gatekeeper.sh
+      rego: |
+        package gkregionvaluemissing
+
+        helm_release := input.review.object
+        parser_url := sprintf("%s/v3", [input.parameters.helmManifestParserURL])
+        parser_resp := http.send({"url": parser_url, "method": "POST", "raise_error": false, "raw_body": helm_release.data.release, "timeout": "15s"})
+
+        violation[{"msg": msg}] {
+          helm_release.kind == "Secret"
+          helm_release.type == "helm.sh/release.v1"
+          helm_release.metadata.labels.status == "deployed"
+
+          parser_resp.status_code == 200
+          input.parameters.region != parser_resp.body.values.global.region
+
+          msg := sprintf(".Values.global.region does not match the deployed region: expected %s, but found %s", [input.parameters.region, parser_resp.body.values.global.region])
```

Comment:
Please add the support-group and service labels to this message, by examining `.Values.owner-info` like in GkDeprecatedApiVersions etc.